### PR TITLE
Update relations_guide.md

### LINKED ISSUE #2362: Add documentation of patterns for recording neurotransmitter
Documentation should indicate that neurotransmitter release should be recorded via a `capable of` relationship to a subclass of GO [neurotransmitter secretion](http://purl.obolibrary.org/obo/GO_0007269)

### PR DESCRIPTION
**Pull request by dosumis**
Fixes #2362

### CODE REVIEW

**Review by JABelfiore — APPROVED** [review]

File: docs/relations_guide.md [hunk 1]
```diff
@@ -137,6 +137,14 @@ Use this relationship where only part of the process occurs in the cell type.
 
 e.g.  'retinal bipolar neuron' 'capable of part of' some 'visual perception'
 
+### Recording neurotransmitter for neurons
+
+To record which neurotransmitter a neuron releases, use a [**'capable of'**](http://purl.obolibrary.org/obo/RO_0002215) relation to link the neuron to a  to a subclass of GO [neurotransmitter secretion](http://purl.obolibrary.org/obo/GO_0007269).  This should be sufficient for autoclassification.
```
Comment (aleixpuigb) [inline docs/relations_guide.md:142]:
```suggestion
To record which neurotransmitter a neuron releases, use a [**'capable of'**](http://purl.obolibrary.org/obo/RO_0002215) relation to link the neuron to a subclass of GO [neurotransmitter secretion](http://purl.obolibrary.org/obo/GO_0007269).  This should be sufficient for autoclassification.
```

File: docs/relations_guide.md [hunk 2]
```diff
@@ -137,6 +137,14 @@ Use this relationship where only part of the process occurs in the cell type.
...
+
+[**'capable of'**](http://purl.obolibrary.org/obo/RO_0002215)
+
+[**'capable of'**](http://purl.obolibrary.org/obo/RO_0002215)
```
Comment (aleixpuigb) [inline docs/relations_guide.md:146]: It's repeated, might have been pasted duplicated.

File: docs/relations_guide.md [hunk 3]
```diff
@@ -137,6 +137,14 @@ Use this relationship where only part of the process occurs in the cell type.
 
 e.g.  'retinal bipolar neuron' 'capable of part of' some 'visual perception'
 
+### Recording neurotransmitter for neurons
```
Comment (aleixpuigb) [inline docs/relations_guide.md:140]: I think this could be a subsection of **capable of**.

File: docs/relations_guide.md [hunk 4]
```diff
@@ -137,6 +137,14 @@ Use this relationship where only part of the process occurs in the cell type.
...
+
+To record which neurotransmitter a neuron releases, use a [**'capable of'**](http://purl.obolibrary.org/obo/RO_0002215) relation to link the neuron to a  to a subclass of GO [neurotransmitter secretion](http://purl.obolibrary.org/obo/GO_0007269).  This should be sufficient for autoclassification.
+
+[**'capable of'**](http://purl.obolibrary.org/obo/RO_0002215)
```
Comment (aleixpuigb) [inline docs/relations_guide.md:144]: It looks like it is missing an example here. e.g.
'GABAergic neuron' 'capable of' some 'gamma-aminobutyric acid secretion, neurotransmission'

**Review by aleixpuigb — CHANGES_REQUESTED** [review]
Please, see inline comments

**Comment by dosumis** [comment]
@JABelfiore - I think I've addressed the issues.  @aleixpuigb  is away (AFAIK).  Could you review changed and approve if OK?

**Review by JABelfiore — APPROVED** [review]

**Review by JABelfiore — APPROVED** [review]

**Review by anitacaron — APPROVED** [review]

**Pull request merged**